pkg/logs/input/listener/udp_test.go

```diff
@@ -38,28 +38,62 @@ func TestUDPShouldReceiveMessage(t *testing.T) {
 	listener.Stop()
 }
 
-func TestUDPShouldProperlyTruncateTooBigMessages(t *testing.T) {
+func TestUDPShouldProperlyTruncateBigMessages(t *testing.T) {
 	pp := mock.NewMockProvider()
 	msgChan := pp.NextPipelineChan()
-	listener := NewUDPListener(pp, config.NewLogSource("", &config.LogsConfig{Port: udpTestPort}), 100)
+	listener := NewUDPListener(pp, config.NewLogSource("", &config.LogsConfig{Port: udpTestPort}), defaultFrameSize)
 	listener.Start()
 
 	conn, err := net.Dial("udp", fmt.Sprintf("localhost:%d", udpTestPort))
 	assert.Nil(t, err)
 
 	var msg message.Message
 
-	fmt.Fprintf(conn, strings.Repeat("a", 80)+"\n")
+	fmt.Fprintf(conn, strings.Repeat("a", defaultFrameSize-100)+"\n")
+	msg = <-msgChan
+	assert.Equal(t, strings.Repeat("a", defaultFrameSize-100), string(msg.Content()))
+
+	fmt.Fprintf(conn, strings.Repeat("a", defaultFrameSize)+"\n")
+	msg = <-msgChan
+	assert.Equal(t, strings.Repeat("a", defaultFrameSize), string(msg.Content()))
+
+	fmt.Fprintf(conn, strings.Repeat("a", defaultFrameSize+100)+"\n")
 	msg = <-msgChan
-	assert.Equal(t, strings.Repeat("a", 80), string(msg.Content()))
+	assert.Equal(t, strings.Repeat("a", defaultFrameSize), string(msg.Content()))
+
+	fmt.Fprintf(conn, strings.Repeat("a", defaultFrameSize-200)+"\n")
```

Comment:
what's the difference between this test and the first one?

Reply:
it's just to ensure that the next message won't be mixed with the rest of the content of the previous message

Reply:
worth a comment then